### DISCUSSION TITLE
Is it possible to write description on PSL?

### DISCUSSION THREAD

**Discussion by hyochan** [prisma/prisma#10754] [Q&A] [answered]
<img width="504" alt="Screen Shot 2021-12-18 at 4 47 46 PM" src="https://user-images.githubusercontent.com/27461460/146633756-31fec79e-706c-448a-8110-57defaa4d0db.png">

It'd be good if we can add @description tag

**Comment by janpio** [comment]
Where and how would that `@description` be used?

**Reply by hyochan** [reply]
Good question. Even in pure DBMS, we can add `comment` like below.

<img width="672" alt="Screen Shot 2021-12-21 at 10 45 23 AM" src="https://user-images.githubusercontent.com/27461460/146856757-4789c473-775f-49a2-867e-905334adf72f.png">

This can be transferred to the description in `nexus` or `graphql` either without any further rewriting.
<img width="408" alt="Screen Shot 2021-12-21 at 10 47 37 AM" src="https://user-images.githubusercontent.com/27461460/146856964-ddb96281-069e-4433-b1f1-d1965bee3424.png">

Does this make sense @janpio ?

**Reply by janpio** [reply, marked as answer]
Yes of course.

There are a few feature requests related to this already: https://github.com/prisma/prisma/issues/8703

At the same time we already have schema comments, which generators could use to output into the generated artifacts: https://www.prisma.io/docs/concepts/components/prisma-schema#comments

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
